electron/src/calling/PictureInPictureCall.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {getNewWindowOptions} from '../window/WindowUtil';
+
+const PICTURE_IN_PICTURE_CALL_FRAME_NAME = 'WIRE_PICTURE_IN_PICTURE_CALL';
+
+export const isPictureInPictureCallWindow = (frameName: string): boolean => {
+  return frameName === PICTURE_IN_PICTURE_CALL_FRAME_NAME;
+};
+
+export const getPictureInPictureCallWindowOptions = (): Electron.BrowserWindowConstructorOptions => {
+  return getNewWindowOptions({
+    width: 1026,
+    height: 829,
```

Comment:
Not a fan of the fixed values here. Is there a way to get these values from the open window?

Reply:
Not sure what do you mean? Here we specify, what is the initial size of the newly opened window. User can then resize it to match their needs.

Reply:
I mean, hat we open it in an fixed size. I would rather see us inherit this size from the parent window (like calc(inherit-50px) ). Something like that :D

Reply:
This is a product decision I'd say. This is just the first iteration, it's likely that we will change things here and there

Reply:
Fine for me

Reply:
i tend to agree with adrian somewhat. can we allow the size to go smaller than this? i feel like theres a scenario id want it to be like a small box i could just float somewhere innocuous.

Reply:
Yes, this will be improved in the future, the current designs for MVP v2 don't address smaller sizes.